pkgs/development/python-modules/snowflake-sqlalchemy/default.nix

```diff
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "snowflake-sqlalchemy";
-  version = "1.4.3";
+  version = "1.4.4";
```

Comment:
Is this upgrade needed for the upgrade of `snowflake-connector-python`? The release notes of the two libs did not make that obvious.

Reply:
No but it's a bugfix that can be backported anyway.

Reply:
Thanks for the confirmation, I was not sure if I missed something or not